wp-cli/vip-migrations.php

```diff
@@ -58,12 +58,15 @@ function cleanup( $args, $assoc_args ) {
 
 		// Cleanup options
 		$options = [
-			'jetpack_options',
-			'jetpack_private_options',
 			'vaultpress',
 			'wordpress_api_key',
 		];
 
+		if ( ! defined( 'VIP_JETPACK_SKIP_LOAD' ) || ! VIP_JETPACK_SKIP_LOAD ) {
+			$options[] = 'jetpack_options';
+			$options[] = 'jetpack_private_options';
+		}
+
```

Comment:
I think we should revert this part as it used to be. I.e., we should keep trying to delete these options, otherwise, we may end up in a situation where Jetpack is reactivated and there are production tokens in non-prod DB.